ios/FluentUI/People Picker/ContactView.swift

```diff
@@ -0,0 +1,184 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+open class ContactView: UIView {
+    private struct Constants {
+        static let contactWidth: CGFloat = 70.0
+        static let avatarViewWidth: CGFloat = 70.0
+        static let avatarViewHeight: CGFloat = 70.0
+        static let titleLabelMinimumHeight: CGFloat = 12.0
+        static let spacingBetweenAvatarAndFirstLabel: CGFloat = 13.0
+    }
+
+    // TODO: Set it as the default image (pawn)
+    @objc public var avatarImage: UIImage? {
+        didSet {
+            if let avatarImage = avatarImage {
```

Comment:
If the UIImage is nil, then you won't get into this condition. 
We would want to set the avatar view's image to nil if this property is set to nil.

Reply:
Commit c2dad3000a5dce7e5bd6ba82d38c12cd8bc87ccd addresses this issue.